CI-test/Foo.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Foo.swift
+//  CI-test
+//
+//  Created by Eugen Spinu on 10/01/2021.
+//
+
+import Foundation
+
+struct Foo {
+    let param1: Int
+    let param2: Int
+    
+    func foo() -> Int {
+        _ = foo1()
+        let bla = "unused variable"
+        return param1+param2
+    }
+    func foo1() -> Int {
+        if param1 == 3 {
+            empty()
+        } else if param2 == 233 {
+            if param1 == 11 {
+                if param1 == 11 {
+                    if param1 == 21 {
+                        if param1 == 13 {
+                            if param1 == 11 {
+                                if param1 == 11 {
+                                    
+                                }
+                            }else if param1 == 11 {
+                                if param1 == 11 {
+                                    
+                                }
+                            }
+                        }else if param1 == 11 {
```

Comment:
<table data-meta="generated_by_metrics">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/espinu/iOS-CI-test/blob/1b7cc7631f4c0843c2828e4a6f1fd4c94af4372a/CI-test/Foo.swift#L36"></span>Else and catch should be on the same line, one space after the previous declaration.<br />
<code>statement_position</code> <code>Foo.swift:36</code></td>
    </tr>
  </tbody>
</table>